### DISCUSSION TITLE
Xray(VLESS+Vision+TLS)+Caddy /1_caddy.json 各位大佬，这个配置如何增加 forward_proxy？

### DISCUSSION THREAD

**Discussion by sw19999** [lxhao61/integrated-examples#218] [Q&A] [answered]
也就是N协议。修改了一下没成功。
                  "srvh3": {
                    "listen": [":8888"],
这个端口应该不能用443了吧？换了也不行。应该是配置文件没写对

**Comment by lxhao61** [comment, marked as answer]
极度不推荐！直接添加 NaiveProxy 插件配置，仅能使用 HTTPS 协议连接，且其应用无法防止被封。
推荐参考  [Xray(E+F+H+A)+Caddy(N)](https://github.com/lxhao61/integrated-examples/tree/main/Xray(E%2BF%2BH%2BA)%2BCaddy(N)) 配置使用。

**Reply by sw19999** [reply]
感谢大佬回复。一开始我也是想用这个。但是集成太多协议，脚本不熟悉，尝试只留下
E=VLESS+Vision+TLS
A=VLESS+mKCP+seed
N=NaiveProxy
这三个协议。我再尝试一下。谢谢大佬
